Source/Meadow.Units/ApparentPower.cs

```diff
@@ -0,0 +1,412 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics.Contracts;
+using System.Runtime.InteropServices;
+using Meadow.Units.Conversions;
+
+namespace Meadow.Units
+{
+    /// <summary>
+    /// Represents Apparent Power 
+    /// </summary>
+    [Serializable]
+    [ImmutableObject(true)]
+    [StructLayout(LayoutKind.Sequential)]
+    public struct ApparentPower :
+        IComparable, IFormattable, IConvertible,
+        IEquatable<double>, IComparable<double>
+    {
+        /// <summary>
+        /// Creates a new `ApparentPower` object.
+        /// </summary>
+        /// <param name="value">The ApparentPower value.</param>
+        /// <param name="type">kilometers meters per second by default.</param>
+        public ApparentPower(double value, UnitType type = UnitType.VoltAmpheres)
+        {
+            Value = ApparentPowerConversions.Convert(value, type, UnitType.VoltAmpheres);
+        }
+
+        /// <summary>
+        /// Creates a new `ApparentPower` object from an existingApparentPower object
+        /// </summary>
+        /// <param name="reactivePower"></param>
+        public ApparentPower(ApparentPower reactivePower)
+        {
+            this.Value = reactivePower.Value;
+        }
+
+        /// <summary>
+        /// Internal canonical value.
+        /// </summary>
+        private readonly double Value;
+
+        /// <summary>
+        /// The type of units available to describe ApparentPower.
+        /// </summary>
+        public enum UnitType
+        {
+            /// <summary>
+            /// Gigavolt Amphere
+            /// </summary>
+            GigavoltAmpheres,
```

Comment:
`Amperes`, not `Ampheres`

I suggest find/replace. :)

Reply:
done :)